verifact/forum/models.py

```diff
@@ -10,3 +11,22 @@ class Question(models.Model):
 
     def __str__(self):
         return self.text
+
+
+class Answer(models.Model):
+    class AnswerTypes(models.IntegerChoices):
+        TRUE = 1, _("True")
+        FALSE = 2, _("False")
+        OTHER = 3, _("Other")
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    answer = models.IntegerField(choices=AnswerTypes.choices)
+    text = models.CharField(max_length=2048)
+    citation_url = models.CharField(max_length=2048)
+    citation_title = models.CharField(max_length=2048)
+    credible_count = models.IntegerField()
+    not_credible_count = models.IntegerField()
+    question_id = models.ForeignKey(Question, on_delete=models.CASCADE)
```

Comment:
You can name this `question`. Django will allow us to say something like `Answer.objects.first().question`, which will provide a cleaner syntax. Behind the scenes, Django creates a `question_id` column though.

Reply:
You can also add `related_name="answers"` to tell Django what name to serve results under. Otherwise, the default plural from the GraphQL api will be `answerSet`, which I think is a bit clunky.